render/wgpu/src/backend.rs

```diff
@@ -515,16 +541,22 @@ impl<T: RenderTarget + 'static> RenderBackend for WgpuRenderBackend<T> {
         &mut self,
         handle: &BitmapHandle,
         bitmap: Bitmap,
-        region: PixelRegion,
+        mut region: PixelRegion,
     ) -> Result<(), BitmapError> {
         let texture = as_texture(handle);
 
+        let mut bitmap = bitmap.to_rgba();
+        if self.clamp_bitmap(&mut bitmap) {
```

Comment:
Since we perform clamping when the bitmap is initially registered, will this ever do anything? The incoming `bitmap` can't be larger than the existing `BitmapHandle`, or the `write_texture` will fail.

Reply:
Yes, things panic without this.
If a game registers a bitmap of 1000 and we internally resize it to 500, the game still thinks it's 1000 and will update it with a 1000 bitmap, which is wrong for the actual texture we're storing.